test/test_rentlinx.rb

```diff
@@ -1,8 +1,23 @@
 require 'minitest/autorun'
 require 'rentlinx'
+require 'vcr'
 
 # Test for Rentlinx module
-class TestRentlinx < MiniTest::Unit::TestCase
+class TestRentlinx < MiniTest::Test
+  def setup
+    # We use hide the real values using VCR
+    @auth_key = ENV['RENTLINX_AUTH_KEY'] || '<AUTH_KEY>'
+    @password = ENV['RENTLINX_PASSWORD'] || '<PASSWORD>'
+    @username = ENV['RENTLINX_USERNAME'] || '<USERNAME>'
+    @site_url = ENV['RENTLINX_SITE_URL'] || 'http://localhost'
```

Comment:
Not sure how I feel about this. Seems like something we'll be putting at the top of every test file. Could it be moved out to `test_helper.rb`? Not important right now but it's a fair amount of boilerplate.

Also, I think it might be easier to load these values in from a yaml file, we could create a default one, commit it, then gitignore it and change it to the correct values. That way we don't have to add _more_ secrets to our .profiles, or type `RENTLINX_AUTH_KEY=412341 RENTLINX_PASSWORD=asdfasd ... rake` whenever we want to re-record.